src/mixxxapplication.cpp

```diff
@@ -1,10 +1,24 @@
 #include <QtDebug>
 #include <QTouchEvent>
-
 #include "mixxxapplication.h"
 #include "control/controlproxy.h"
 #include "mixxx.h"
 
+// When linking Qt statically on Windows we have to Q_IMPORT_PLUGIN all the
+// plugins we link in build/depends.py.
+#ifdef QT_NODLL
```

Comment:
don't we need to enforce the "on windows" part of your comment here ?
How are imageformats plugins under Linux/MacOS ?